src/Cli/dotnet/Parser.cs

```diff
@@ -118,6 +118,19 @@ public static Command GetBuiltInCommand(string commandName)
             .EnableLegacyDoubleDashBehavior()
             .Build();
 
+         class HelpResult : IInvocationResult
```

Comment:
This HelpResult is a copy/paste of the HelpResult type in the command-line-api repo [here](https://github.com/dotnet/command-line-api/blob/main/src/System.CommandLine/Help/HelpResult.cs). This type isn't internal so it must be reproduced in full here.